source/includes/usage-examples/index-code-examples.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+$database = $client->sample_db;
+$collection = $database->sample_coll;
```

Comment:
It looks like `sample-app.php` was changed to use:

```php
$collection = $client->selectCollection('<database>', '<collection>');
```

Did you mean to do the same here for consistency? It doesn't look like this content is presented to the user at all since it's outside of start/end comments.

Reply:
This is not visible to users, so it didn't seem necessary to update